traitsui/qt4/datetime_editor.py

```diff
@@ -15,60 +15,79 @@
 """
 
 
-import datetime
-
-from pyface.qt import QtGui
+from pyface.qt import QtGui, qt_api
 from pyface.qt.QtCore import QDateTime
-from traits.api import Instance
+from traits.api import Datetime, on_trait_change
 
 from .editor import Editor
 from .editor_factory import ReadonlyEditor as BaseReadonlyEditor
 
 
+
 class SimpleEditor(Editor):
     """ Simple Traits UI time editor that wraps QDateTimeEdit.
     """
 
     #: the earliest datetime allowed by the editor
-    minimum_datetime = Instance(datetime.datetime)
+    minimum_datetime = Datetime
```

Comment:
This keeps things synchronized if Traits changes the definition of the `Datetime` trait, so it is preferable.

It's not cyclic, because the `datetime_editor` is actually the Datetime editor _factory_, and this is the actual editor (plus there's a couple of layers of delayed imports in between the trait and the Qt editor).